src/app/page/page.component.ts

```diff
@@ -24,7 +24,12 @@ namespace Page {
           this.previousWidth = this.$window.innerWidth;
         });
       });
-      this.$rootScope.$on(CLOSE_MAIN_NAV_EVENT, () => this.isNavOpen = false);
```

Comment:
Found one more issue. This **must not** be removed. This line is where the page accepts the close event from other components. Removing it will disable this function.